### DISCUSSION TITLE
Self-destruct button for Codespaces

### DISCUSSION THREAD

**Discussion by jtmaveryk** [community/community#61983] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I'd like to see the ability to self-destruct a GitHub code space from internally inside the Codespace (File -> Self-destruct) & command palette, rather than having to return to the Codespaces page or repo to destroy it.

**Comment by h7x4** [comment]
That would be cool yeah.

For the record and ease of anyone reading this thread, these are the current options:

![image](https://github.com/community/community/assets/14929991/1a43b9cc-b713-44a1-9e8e-ae5d477d9abf)
